ql/src/test/queries/clientpositive/translated_external_rename3.q

```diff
@@ -1,26 +1,25 @@
 set metastore.metadata.transformer.class=org.apache.hadoop.hive.metastore.MetastoreDefaultTransformer;
-set metastore.metadata.transformer.location.mode=force;
```

Comment:
this is also working in force mode also. Can you please give some info about why you had to change it to seqprefix?

Reply:
this was a mistake - I've clobbered this existing test too much; I've moved the new testcase into a new qfile.

the location mode shouldn't matter for this test at all.